docs/chapters/sdk/template-service.rst

```diff
@@ -0,0 +1,19 @@
+Template Service
+================
+
+A template service serves as an example CMake_ project for service developers that should be used as a starting point while creating platform services. To acquire the template service source code the user needs to fetch the code from github.
+
+.. code-block:: bash
+
+    git clone https://github.com/Pelagicore/template-service.git
+
+In order to compile the template service the user needs to use the CMake build system.
```

Comment:
It does, I was thinking of how we often do it where we tell the user the name of the debian package like here: https://pelagicore.github.io/software-factory/chapters/baseplatform/index.html#dependencies-for-debian-users